src/theme/popover/ods-chart-popover-definitions.ts

```diff
@@ -237,9 +252,17 @@ class BOOSTED4_Definition extends ODSChartsPopoverDefinitionWithRenderer {
       container: 'body',
       title: title,
       content: htmlContent,
+      customClass: '',
     });
-    elt.chart = popover;
+    elt.chartPopover = popover;
 
+    if (!enterable) {
+      window.setTimeout(() => {
+        try {
+          elt.chartPopover.tip.style.pointerEvents = 'none';
```

Comment:
This looks more like a workaround, but I can't find another right now, let's go with it for now!

Reply:
I did not found any class in boosted 4 to do it :(